Mathlib/Algebra/Field/Defs.lean

```diff
@@ -47,61 +45,83 @@ a `GroupWithZero` lemma instead.
 field, division ring, skew field, skew-field, skewfield
 -/
 
+-- `NeZero` should not be needed in the basic algebraic hierarchy.
+assert_not_exists NeZero
+
+-- Check that we have not imported `Mathlib.Tactic.Common` yet.
+assert_not_exists Mathlib.Tactic.LibrarySearch.librarySearch
```

Comment:
seems like a strange way to test for this, especially because something like that might move to `Std` at some point. (EDIT: nevermind, I see you just pulled this from the bottom of the file. Is there a reason for moving it?)

Reply:
The reason is that the content of a file doesn't change whether an `assert_not_exists` in it succeeds or not, so we might as well put it as early as possible.

I believe @semorrison started by putting them at the end, but eventually decided to put them first.